### DISCUSSION TITLE
Will there be any out of the box support for Mongo-DB ?

### DISCUSSION THREAD

**Discussion by aneessh18** [dropwizard/dropwizard#5816] [Ideas]
Currently Foundation-DB is supported by Dropwizard, is there any intention of supporting Mongo-DB in the same capacity ?

**Comment by joschi** [comment]
There's a Dropwizard module for MongoDB from the @washingtonpost:

https://github.com/washingtonpost/dropwizard-mongo-module

**Reply by aneessh18** [reply]
Above project doesn't have native Morphia support. It would be great if there is native support for Morphia and Mongo DB in Dropwizard.

**Reply by joschi** [reply]
You could contribute to the existing project or fork it and implement Morphia support. 😉 

There are currently no plans to officially support a module for MongoDB unless one is being donated and maintained by community members.
